src/fides/api/ops/schemas/connection_configuration/connection_secrets_email.py

```diff
@@ -10,7 +12,7 @@ class EmailSchema(ConnectionConfigSecretsSchema):
     """Schema to validate the secrets needed for the EmailConnector"""
 
     to_email: str
-    test_email: Optional[str]  # Email to send a connection test email
+    test_email: Optional[EmailStr]  # Email to send a connection test email
```

Comment:
Pretty much all the connectors have string inputs - the sass connectors too - The Saas ones aren't as obvious because their schemas are generated dynamically https://github.com/ethyca/fides/blob/main/src/fides/api/ops/schemas/connection_configuration/connection_secrets_saas.py#L99

The connector_params and the external_references become string inputs 
https://github.com/ethyca/fides/blob/main/data/saas/config/firebase_auth_config.yml#L8